src/mpl2/README.md

```diff
@@ -109,6 +109,23 @@ place_macro
 | `-location` | The lower left corner of the macro in microns. |
 | `-orientation` | The orientation according to odb. If nothing is specified, defaults to `R0`.  We only allow `R0`, `MY`, `MX` and `R180`.  |
 
+### Set Macro Guidance Region
+
+Command for setting guidance regions for macros.
+
+```tcl
+set_macro_guidance_region
+    -macro_name macro_name
+    -region region
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-macro_name` | The name of a macro of the design. |
+| `-region` | The lower left corner and upper right corner {lx ly ux uy} of the region in microns. |
```

Comment:
Clarify: within which the macro must fit.

Reply:
https://github.com/joaomai/OpenROAD/blob/49ce3f117a13f28a3ae92fe4a214735927ba3bfa/src/mpl2/src/rtl_mp.cpp#L203-L217

@oharboe Is this what your asking about?

Reply:
No, if the macros for a guidance region cant possibly fit, it would be good with an early clear error, rather than a more obscure error much later.

Reply:
The placer tries to maximize overlap with the region.  If the region is smaller then it says you are willing to accept any overlapping placement (ie there is some slop).

Reply:
> The placer tries to maximize overlap with the region. If the region is smaller then it says you are willing to accept any overlapping placement (ie there is some slop).

Seems useful to know... Clarify in docs?

Reply:
I'll update the command description.